src/extension.ts

```diff
@@ -117,7 +118,8 @@ export async function attach(context: vscode.ExtensionContext, attachUrl?: strin
             // Auto connect to found target
             telemetryReporter.sendTelemetryEvent("command/attach/devtools", telemetryProps);
             const runtimeConfig = getRuntimeConfig(config);
-            DevToolsPanel.createOrShow(context, telemetryReporter, targetWebsocketUrl, runtimeConfig);
+            const stringMap = await getLocalizedStrings(context.extensionPath);
+            DevToolsPanel.createOrShow(context, telemetryReporter, targetWebsocketUrl, runtimeConfig, stringMap);

```

Comment:
Should we just move the getLocalizedStrings call into the createOrShow function?